### DISCUSSION TITLE
Stutter and Lag - started recently - Macbook pro 16''

### DISCUSSION THREAD

**Discussion by gabrieldabbah** [ppy/osu#21955] [Q&A]
### Type

Performance

### Bug description

MacBook Pro 16'' M1 PRO 16GB
macOS Ventura
3440x1440 7:3 screen

For the last couple of weeks, the game has kept stuttering (freezes) every minute or so for about ˜200-500ms. I literally cannot FC a single map because of it. These stutters did not happen before the update on the 8th of December. I thought they would go away in time. They didn't.

Also, I have about 300fps during gameplay. I used to have more than 1000 before the update. What happened?

I tried multiple settings. Nothing changes the fps and the stutters.

### Version
osu!lazer
all versions since december 8

### Logs

There is no "osu" folder at  ~/Library/Application Support/osu/logs

**Comment by bdach** [comment]
First of all please read and follow the steps here: https://osu.ppy.sh/wiki/en/Performance_troubleshooting. Guide is for stable but it still pretty much applies to lazer too.

If nothing from there helps, then we're going to need details about this "stutter and lag". Like a video, preferably with fps counter or even frame graph showing (ctrl-f11 twice to display).

**Comment by peppy** [comment]
This can happen on macOS when any other app is doing something weird. As above, providing the frame graph video/images may will help us tell you what the issue is, but your best bet is to trial-and-error close all other apps on your system until the issue disappears.
